ddtrace/debugging/_expressions.py

```diff
@@ -67,6 +68,8 @@ def _make_function(ast, args, name):
         raise ValueError("Invalid predicate: %r" % ast)
 
     instrs = compiled + [Instr("RETURN_VALUE")]
+    if sys.version_info >= (3, 11):
+        instrs.insert(0, Instr("RESUME", 0))
```

Comment:
where I can read more about this?

Reply:
https://docs.python.org/3/library/dis.html#opcode-RESUME